scripts/build.js

```diff
@@ -10,6 +10,8 @@ import { stdoutColors } from "../util/terminal.js";
 const require = createRequire(import.meta.url);
 const dirname = path.dirname(fileURLToPath(import.meta.url));
 const watch = process.argv[2] === "--watch";
+const debug = process.argv[2] === "--debug";
+const release = process.argv[2] === "--release";
```

Comment:
Should there perhaps be a default here if no build type is specified? Seems that right now this would result in `debug = false` and `release = false`. For example:

```suggestion
const debug = process.argv[2] === "--debug";
const release = process.argv[2] === "--release" || !debug;
```